hw/ip/flash_ctrl/data/flash_ctrl_testplan.hjson

```diff
@@ -24,116 +24,112 @@
+            interface.
             '''
       milestone: V2
-      tests: []
+      tests: ["flash_ctrl_read_direct_b2b"]
     }
     {
-      name: intr_enabled
+      name: host_controller_arb
       desc: '''
-            Sanity + enable all interrupts and verify correctness of assertion and de-assertion of
-            each pin. Replace polling with waits on interrrupts.
+            Perform back-to-back operations via Software Interface and read of seed materials
+            via Hardware Interface in order to test arbitration within Flash Protocol Controller.
             '''
       milestone: V2
-      tests: []
-    }
+      tests: ["flash_ctrl_controller_arb"]
+    }    
     {
-      name: host_ctrl_arb
+      name: full_memory_access
       desc: '''
-            Continuously read data from host interface while randomly programming or reading from
-            the flash controller. Ensure the expected bandwidths are seen on both interfaces. Also,
-            initiate reads from the host to all banks back and forth to ensure the responses arrive
-            in-order.
+            Entire memory is accessed by Controller and directly by Host. In addition, Data
+            partitions can be directly read by Software(Flash controller) and hardware hosts,
+            while Info partitions can be read only by the Flash controller.
             '''
       milestone: V2
-      tests: []
+      tests: ["flash_ctrl_full_mem_access"]
     }
     {
-      name: host_read_collide
+      name: host_arb
       desc: '''
-            Read LSB bus word via host and program or read the MSB bus word at the same time. This
-            will cause the internal flash word wide buffer to be updated. Then read the MSB bus word
-            within the same flash data word - it should read back the updated data, proving that the
-            internal buffer was cleared from the previous step.
+            Test arbitration within Flash Physical Controller by reading from both interfaces at
+            the same time. Perform continuously read data from host interface while, at the same
+            time, reading from the flash controller is in process. Perform parallel reading at
+            different addresses and on the same address. Expect that operations are successfully
+            executed.
             '''
       milestone: V2
-      tests: []
+      tests: ["flash_ctrl_host_arb"]
     }
     {
-      name: en_mp_regions
+      name: host_interleave
       desc: '''
-            Protect randomized and legal ranges of data and info partitions and ensure that the
-            corresponding read / program / erase enable bits are working. TBD - future support for
-            sub-page protection may be added - that will be tracked as a part of this test as well.
+            At same time, perform two read operations and the same time via hardware host and via
```

Comment:
o looks like this is already covered!

Reply:
Ok. This test is exactly meant to exercise arbitration cases which are not typical for the user.

Reply:
just a note here that the hardware host (assuming that's what you're talking about) behavior cannot be arbitrarily controlled (it is "fixed").